### PR TITLE
zerofree: install manpage and COPYING file

### PR DESCRIPTION
**Pull request by Moredread**
###### Motivation for this change

Install COPYING and manpage from Debian/Ubuntu.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/filesystems/zerofree/default.nix [hunk 1]
```diff
@@ -9,12 +14,14 @@ stdenv.mkDerivation rec {
...
 
-  buildInputs = [ e2fsprogs ];
+  buildInputs = [ e2fsprogs installShellFiles ];
 
   installPhase = ''
-    mkdir -p $out/bin
+    mkdir -p $out/bin $out/share/zerofree
     cp zerofree $out/bin
-'';
+    cp COPYING $out/share/zerofree
```
Comment (SuperSandro2000) [inline pkgs/tools/filesystems/zerofree/default.nix:22]:
```suggestion
    cp COPYING $out/share/zerofree/COPYING
```

Did you mean this? Also I don't feel comfortable to fetch a manpage from Ubuntu.

**Comment by SuperSandro2000** [comment]
@moredread please resolve the merge conflict.

**Comment by Moredread** [comment]
@SuperSandro2000 there is no official page for this package. Alternatively we could maybe add a ubuntu-manpages and debian-manpages derivation and link from those. I'm not sure though how large the man pages are that are provided by Debian and Ubuntu and if we want to add a large dependency only for the man page.

Maybe this is a larger issue... When I get around to it I'll write something up und Discourse.

Are you still interested in this PR?

**Comment by SuperSandro2000** [comment]
> Are you still interested in this PR?

I am not interested in most PRs personally. If you think this is a good addition go for it.

**Comment by jtojnar** [comment]
Alternately, you could try to work with upstream and get it to accept the manpage from Debian. We did that for other packages successfully in the past ([example](https://gitlab.gnome.org/GNOME/libnotify/-/merge_requests/6)).

**Comment by Moredread** [comment]
I've resolved the merge conflict.

**Pull request merged**